helix-term/Cargo.toml

```diff
@@ -61,6 +61,8 @@ toml = "0.7"
 
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
+zbus = { version = "3.13" }
+zvariant = { version = "3.13" }
```

Comment:
This pulls in hundreds of dependencies just to read a single value. Could this be done by a wrapper script on the user side that's executed when the editor starts?

Reply:
Yes that's quite unfortunate, though I think they don't have a big impact on the binary size I'll test that. I'm going to add functionality that watches the dbus as well so a wrapper script won't cut it.
I am thinking though maybe there's a 'cheaper' way to interact with the dbus by using cli tools like `dbus-send`